storage/innobase/xtrabackup/doc/source/advanced/page_tracking.rst

```diff
@@ -0,0 +1,124 @@
+.. _page_tracking:
+
+=======================================
+Incremental backup using page tracking
+=======================================
+
+This feature is *tech preview* quality.
+
+Percona XtraBackup 8.0.27 adds support for the page tracking functionality for `incremental backup <https://www.percona.com/doc/percona-xtrabackup/8.0/backup_scenarios/incremental_backup.html>`__ . 
+
+To create an incremental backup with page tracking, Percona XtraBackup uses the MySQL mysqlbackup component. This component looks in the InnoDB data files for pages that have been modified since the last backup and stores these pages in a file under the server's datadir. Then Percona XtraBackup copies only those changed pages. This operation removes the need to scan all of the pages in the database. If the majority of pages have not been modified, the page tracking feature can improve the speed of incremental backups.
+
+Install the component
+---------------------
+
+To start using the page tracking functionality, do the following steps:
+
+1. Install the mysqlbackup component and enable it on the server: 
+
+  .. code-block:: mysql
+
+     $ INSTALL COMPONENT "file://component_mysqlbackup";
+
+2. Check whether the mysqlbackup component is installed succesfuly:
+
+  .. code-block:: bash
+
+     SELECT COUNT(1) FROM mysql.component WHERE component_urn='file://component_mysqlbackup';
+
+Start page tracking
+-------------------
+
+After the mysqlbackup component is loaded and active on the server, start the page tracking:
+
+  .. code-block:: mysql
+
+     $ SELECT mysqlbackup_page_track_set(true);
+
+.. note::
+
+   Percona XtraBackup processes all the tracked pages in memory. If Percona XtraBackup does not have enough available memory to process all the tracked pages, Percona XtraBackup throws an error and exits. For example, if an incremental backup uses 200GB, Percona XtraBackup can additionally use about 100MB of memory to store the page tracking data.
```

Comment:
```suggestion
   Percona XtraBackup processes list of all the tracked pages in memory. If Percona XtraBackup does not have enough available memory to process  list of all the tracked pages, Percona XtraBackup throws an error and exits. For example, if an incremental backup uses 200GB, Percona XtraBackup can additionally use about 100MB of memory to store the page tracking data.
```

Reply:
moves this notes at the end. just before the known issue.

Reply:
I've moved this note to the Using page tracking section according to the doc structure.